src/common/components/Table.tsx

```diff
@@ -1,4 +1,4 @@
-import { useEffect, useMemo, useState } from 'react';
+import { useEffect, useMemo, useRef, useState } from 'react';
```

Comment:
Note to Future David: Have brain thoughts about more documentation and more intuitive api.